core/cmd/hoverfly/main.go

```diff
@@ -179,9 +180,18 @@ func init() {
 func main() {
 	hoverfly := hv.NewHoverfly()
 
+	dir, err := os.Getwd()
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+
```

Comment:
Yes. But getting the cwd can return an error (e.g. https://manned.org/sys_getcwd/ef67ad78).

Reply:
But I'll double check if we can ignore the error and keep the `dir` variable to be an empty string. Thus we just delay the error handling and it'll occur when a bodyFile is not able to be read from in case it contains a relative path.

Reply:
what I meant is: 
```
ioutil.ReadFile(filepath.Join("", pair.Response.GetBodyFile()))
```
should read the file under the current working directory, right? 
so setting the `response-body-files-path` to "" should be equivalent to setting it to `os.Getwd()`?

Reply:
Oh, yes. I used the `os.Getwd` to show the current dir in `--help` information, otherwise there will be an empty string there.